### PR TITLE
Advanced image augmentation

### LINKED ISSUE #32: Apply Parallel light for Enhanced Image Augmentation
Applying brightness decay function and Spotlight functions for advanced image augmentation methods. This would help the model to learn and predict in different lightening conditions

### PR DESCRIPTION
**Pull request by humbletechy**
Fixes #32 

Applied Parallel light and Spot Light for better Image Augmentation for better predictions

### CODE REVIEW

File: visionppi/.idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -3,6 +3,115 @@
     <JetCodeStyleSettings>
       <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
     </JetCodeStyleSettings>
+    <codeStyleSettings language="XML">
```
Comment (yashk2000) [inline visionppi/.idea/codeStyles/Project.xml]: Please revert this change. Changes to any file in the .idea folder should never be pushed.

Comment (humbletechy) [reply visionppi/.idea/codeStyles/Project.xml]: Sorry I forgot to git ignore, correcting it

**Comment by yashk2000** [comment]
https://github.com/openMF/ppi-vision/pull/97
